certified-connectors/PDFco/apiDefinition.swagger.json

```diff
@@ -0,0 +1,727 @@
+    "version": "1.0",
+    "contact": {
+      "email": "plugins@bytescout.com",
+      "name": "PDF.co Support",
+      "url": "https://support.bytescout.com"
+    }
+  },
+  "x-ms-connector-metadata": [
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://pdf.co/"
+    },
+    {
+      "propertyName": "Privacy policy",
+      "propertyValue": "https://pdf.co/pages/legal/privacy"
+    },
+    {
+      "propertyName": "Categories",
+      "propertyValue": "Productivity;Data"
+    }
+  ],
+  "host": "api.pdf.co",
+  "basePath": "/",
+  "schemes": ["https"],
+  "consumes": [
+    "application/json",
+    "application/x-www-form-urlencoded"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/v1/pdf/convert/from/html": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "OK, success",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "url": {
+                  "type": "string",
+                  "description": "",
+                  "title": "url",
+                  "x-ms-visibility": "important"
+                },
+                "pageCount": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "",
+                  "title": "pageCount",
+                  "x-ms-visibility": "important"
+                },
+                "error": {
+                  "type": "string",
+                  "description": "",
+                  "title": "error",
+                  "enum": ["", "true", "false"],
+                  "x-ms-visibility": "important"
+                },
+                "status": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "",
+                  "title": "status",
+                  "x-ms-visibility": "important"
+                },
+                "name": {
+                  "type": "string",
+                  "description": "",
+                  "title": "name",
+                  "x-ms-visibility": "advanced"
+                },
+                "credits": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "",
+                  "title": "credits",
+                  "x-ms-visibility": "advanced"
+                },
+                "duration": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "",
+                  "title": "duration",
+                  "x-ms-visibility": "advanced"
+                },
+                "remainingCredits": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "",
+                  "title": "remainingCredits",
```

Comment:
@hiren-byte strings in camel case are considered non-translatable, please make it a sentence so that it looks like `Remaining credits`. Please look through the titles and make sure they look like valid sentences or start with a capital letter if it's a single word title. More information about strings in English can be found [here](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings).